RPIPs/RPIP-59.md

```diff
@@ -1,90 +0,0 @@
----
-rpip: 59
-title: Deposit Mechanics
-description: Describes the mechanics of Node Operator deposits and validator creation, including standard and express queues.
-author: Valdorff (@Valdorff), knoshua (@knoshua)
-discussions-to: TBD
-status: Draft
-type: Protocol
-category: Core
-created: 2024-03-08
-requires: 42, 43, 44
-tags: tokenomics-2024, tokenomics-content
----
-
-## Abstract
-
-Defines the queue structures that govern validator deposits and the overall deposit process within Rocket Pool. Both a standard and express queue structure are present. Access to the express queue is managed via tickets. A limited number of tickets are provided to all new nodes, and an amount of tickets are provided to existing nodes at the time of the upgrade based on their amount of bonded ETH. The express queue is designed to progress faster than the standard queue.
-
-This proposal also changes the deposit mechanics: In case of a queue, the initial stake transaction happens only once ETH is assigned. This makes it possible to exit from the queue and receive ETH credit up until the validator is dequeued.
-
-## Specification
-
-### Deposit queue specification
-ETH from the deposit pool SHALL be matched with validator deposits from queues as follows:
-- There SHALL be a `standard_queue`
-  - When adding a validator, users MAY place their deposit on the `standard_queue`  
-- There SHALL be an `express_queue`
-  - When adding a validator, users MAY place their deposit on the `express_queue` by spending one `express_queue_ticket`
-- When matching ETH from the deposit pool to queued deposits:
-  - First, ETH SHALL be matched to the oldest deposit in the `express_queue`; this is repeated `express_queue_rate` times
-  - Next, ETH SHALL be matched to the oldest deposit in the `standard_queue`
-  - This sequence SHALL be repeated indefinitely until both queues are empty
-    - If one queue is empty, those matches SHALL be skipped
-- Each node SHALL be provided `express_queue_tickets` equal to `express_queue_tickets_base_provision` (this includes newly created nodes)
-- Each node SHALL be provided additional `express_queue_tickets` equal to `(bonded ETH in legacy minipools)/4` (this will always be zero for newly created nodes)
```

Comment:
Is it worth saying something like: The pDAO can assign additional tickets to all NOs' to cover using this to manage future migrations?

Reply:
I don't think that would be trivial to do, nor do I see a likely scenario for it (and if it's a migration I don't have a scenario for... hard to claim this is the best way).